src/codewhisperer/service/transformByQHandler.ts

```diff
@@ -317,7 +323,7 @@ function getProjectDependencies(modulePath: string): string[] {
         '-Dmdep.copyPom=true',
         '-Dmdep.addParentPoms=true',
     ]
-    const spawnResult = spawnSync(baseCommand, args, { cwd: modulePath, shell: false, encoding: 'utf-8' })
+    const spawnResult = spawnSync(baseCommand, args, { cwd: modulePath, shell: true, encoding: 'utf-8' })
```

Comment:
Discussed above; needed to resolve a customer issue where their Maven executable was not located unless shell is set to true.